app/src/main/java/org/simple/clinic/searchresultsview/PatientSearchView.kt

```diff
@@ -59,11 +80,13 @@ class PatientSearchView(context: Context, attrs: AttributeSet) : RelativeLayout(
   @SuppressLint("CheckResult")
   override fun onFinishInflate() {
     super.onFinishInflate()
-    inflate(context, R.layout.patient_search_view, this)
```

Comment:
How would the layout file be attached if we're removing it from here?

Reply:
We are still inflating the layout. If you look at a couple of lines below. We are using this `PatientSearchViewBinding.inflate(layoutInflater, this)` to inflate the layout.